lib/cpp/access_gpio.py

```diff
@@ -0,0 +1,94 @@
+#
+#   Copyright 2022 Antaris, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# sys.argv[0] = '1' indicates GPIO write and '0' indicates GPIO read
+# sys.argv[1] = Port number
+# sys.argv[2] = Pin number
+
+import time, sys, json
+
+import pylibftdi as ftdi
+
+# Define error code
+g_GPIO_ERROR = -1
+g_SLEEP_TIME_IN_SEC = 1
+g_MASK_BIT_0 = 1
+g_MASK_BYTE = 0xFF
+
+def api_read_gpio(port, pin):
+    try:
+        DeviceName = ftdi.Driver().list_devices()[0][2]  # Assumptioon: single FTDI device connected.
```

Comment:
is there a way to know whether single FTDI device detected or not?

Reply:
This is way to detect ftdi device ... [0][2] index shows 'device name"...